ui/raidboss/data/04-sb/raid/o9s.js

```diff
@@ -185,20 +185,20 @@
     {
       id: 'O9S Fire Phase Tracking',
       regex: / 14:3186:Chaos starts using Blaze/,
-      regexDe: / 14:3186:Chaos starts using /,
-      regexFr: / 14:3186:Chaos starts using /,
-      regexJa: / 14:3186:カオス starts using /,
+      regexDe: / 14:3186:Chaos starts using Flamme/,
+      regexFr: / 14:3186:Chaos starts using Flammes/,
+      regexJa: / 14:3186:カオス starts using ほのお/,
       run: function(data) {
         if (data.phaseType != 'enrage')
           data.phaseType = 'fire';
       },
     },
     {
       id: 'O9S Entropy Spread',
-      regex: /1A:\y{ObjectId}:(\y{Name}) gains the effect of (?:Unknown_640|Entropy) from  for (\y{Float}) Seconds/,
-      regexDe: /1A:\y{ObjectId}:(\y{Name}) gains the effect of (?:Unknown_640|Chaosflammen) from  for (\y{Float}) Seconds/,
-      regexFr: /1A:\y{ObjectId}:(\y{Name}) gains the effect of (?:Unknown_640|Flammes du chaos) from  for (\y{Float}) Seconds/,
-      regexJa: /1A:\y{ObjectId}:(\y{Name}) gains the effect of (?:Unknown_640|混沌の炎) from  for (\y{Float}) Seconds/,
+      regex: / 1A:\y{ObjectId}:(\y{Name}) gains the effect of Entropy from (?:.+|) for (\y{Float}) Seconds/,
```

Comment:
I think `(?:.+|)` could better be written as `.*`

Reply:
will be fixed in next commit